ci/scripts/build-project.sh

```diff
@@ -5,5 +5,10 @@ source $(dirname $0)/common.sh
 repository=$(pwd)/distribution-repository
 
 pushd git-repo > /dev/null
-./gradlew -Dorg.gradle.internal.launcher.welcomeMessageEnabled=false --no-daemon --max-workers=4 -PdeploymentRepository=${repository} build publishAllPublicationsToDeploymentRepository
+if [[ -d /opt/openjdk-secondary ]]; then
+  secondary_java_version=$( ./$(dirname $0)/get-secondary-java-version.sh )
+	./gradlew -Dorg.gradle.internal.launcher.welcomeMessageEnabled=false --no-daemon --max-workers=4 -PdeploymentRepository=${repository} build publishAllPublicationsToDeploymentRepository -PtoolchainVersion=${secondary_java_version} -Porg.gradle.java.installations.auto-detect=false -Porg.gradle.java.installations.auto-download=false -Porg.gradle.java.installations.paths=/opt/openjdk-secondary/
```

Comment:
I'm basically disabling auto-downloads and auto-detection here and specify the location for the toolchain to look for installed JDKs.
I hope this works on Concourse the same way as on my local machine.